### PR TITLE
initial commit; iamspy

### PR DESCRIPTION
**Pull request by dualfade**
please review; thank you

### CODE REVIEW

File: packages/iamspy/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=IAMspy
```
Comment (noraj) [inline packages/iamspy/PKGBUILD:4]:
```suggestion
pkgname=iamspy
```

always lowercase

File: packages/iamspy/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+pkgver=33.9b0173b
+pkgrel=1
+pkgdesc='a library that utilises the Z3 prover to attempt to answer questions about AWS IAM.'
```
Comment (noraj) [inline packages/iamspy/PKGBUILD:7]:
```suggestion
pkgdesc='A library that utilises the Z3 prover to attempt to answer questions about AWS IAM.'
```

**Review by noraj — CHANGES_REQUESTED** [review]

**Comment by dualfade** [comment]
Adjusted

**Review by noraj — APPROVED** [review]
LGTM

**Comment by dualfade** [comment]
Please hold. 

Developer would like to distribute from pypi for release versioning.

Thank you.

**Comment by dualfade** [comment]
Let me check in with the dev. I will update this. Thanks !

**Comment by dualfade** [comment]
Not fixed yet; removing pr for now. Thanks all.

**Pull request closed without merging**